### DISCUSSION TITLE
I would like to create different blocks of questions using extensions, is there any direct way?

### DISCUSSION THREAD

**Discussion by arpit0515** [ueberdosis/tiptap#2637] [General Questions]
I have a custom requirement to fulfill. The need is to have different sets of questions within the tiptap editor and the questions should have a basic node structure like Heading, Sub Heading, Options. I tried using the custom document but as per the functionality, the nodes get deleted when I try to hit backspace till the very first character. Any help is appreciated. I also have the Business Pro License, but have no email address to send the email to.

**Comment by philippkuehn** [comment]
I think we can help better if you could create a code sandbox or something.
